src/main/java/seedu/address/logic/commands/EditClaimCommand.java

```diff
@@ -37,12 +37,13 @@ public class EditClaimCommand extends Command {
             + PREFIX_CLAIM_STATUS + "[NEW_STATUS] "
             + PREFIX_CLAIM_DESC + "[NEW_DESCRIPTION]\n"
```

Comment:
should put square brackets around the prefix as well

Reply:
updated